course/lesson-22-functions-return-values/lesson.tres

```diff
@@ -174,7 +174,7 @@ script = ExtResource( 4 )
 content_id = "res://course/lesson-22-functions-return-values/content-BWNbDGt6.tres"
 title = ""
 type = 0
-text = "To do so, we use a function. It multiplies the cell coordinate by the cell size, adds half the cell size to the product, and returns the result."
+text = "To do so, we use a function. It does two things: first, it multiplies the cell coordinate by the cell size which gets us it's top-left corner posititon in pixels. Then it adds half of it's size to get the center of the cell, and finally returns the result."
```

Comment:
```suggestion
text = "To do so, we use a function. The function does two things:

1. First, it multiplies the cell coordinates by the cell size, which gives us the position of the cell's top-left corner on the screen, in pixels.
2. Then, we add half of the cell size to get the center of the cell.

The function returns the result, allowing us to store it in a variable."
```

Reply:
This needed more thorough changes. When we talk about doing X steps, when possible it's good to use numbered lists, they help with reading. And that even if we didn't have them in the original lessons, making improvements we forgot is always good.

Reply:
This is much better, thanks!